src/lib/fauna-import-writer.js

```diff
@@ -37,10 +37,22 @@ function getFaunaImportWriter(
       this.statusCode = statusCode
     }
   }
+
   const faunaObjectTranslator = new FaunaObjectTranslator(typeTranslations)
 
+  const rateLimiter = new RateLimiterMemory({
+    duration: 1, // seconds
+    points: bytesPerSecondLimit,
+  })
+
   const waitForRateLimitTokens = (tokens, rateLimiter) => {
-    while (!rateLimiter.tryRemoveTokens(tokens)) {
+    const tryToConsumeTokens = () => {
+      return rateLimiter
+        .consume('', tokens)
+        .then(() => true)
+        .catch(() => false)
+    }
+    while (!tryToConsumeTokens()) {
```

Comment:
`const tryToConsumeTokens` is a promise.

Evaluating its truthiness will always evaluate to true.

You need to use async / await to acheive your goal here of waiting until there's enough tokens.

Reply:
A test proving the rate limiter's behavior would have caught this. Its probably worth adding one.

Reply:
a trick i've used to test rate limiters before is something like:

```
start timer
set rate limit to like 1 / s
issue ~3 requests
confirm timer >= 3
```

usually pretty easy to code up.